### DISCUSSION TITLE
Setting Shutter Modes for each Shutter individually

### DISCUSSION THREAD

**Discussion by Gamogamer** [arendst/Tasmota#20913] [Ideas]
I have a project where I have two shutters, one being a DC motor and one being a Servo. I am aware that multiple shutters are possible but they all share the global shuttermode. It would be nice if i could set a different shuttermode for each shutter.

**Comment by stefanbode** [comment]
@Gamogamer : Agree that this would help. The design of the driver is a bit different and there are several situations where a unknown set of different shutters might cause serious issues in timing and so on. Therefore I'm sorry there are no plans to change and the main reason is the required effort of testing after doing a change because of possible side effects.

**Reply by Gamogamer** [reply]
Thank you for the info, i appreciate the explanation
